scripts/microrepl.py

```diff
@@ -112,7 +112,7 @@ class Windows10Console(miniterm.Console):
     }
 
     def __init__(self) -> None:
-        super().__init__()
+        super(miniterm.Console, self).__init__()
```

Comment:
@jdjjm Don't you skip [the logic in the body](https://github.com/pyserial/pyserial/blob/master/serial/tools/miniterm.py#L107) of `miniterm.Console.__init__` that way?